src/views/Restoration/RestorationReviewConfirm.vue

```diff
@@ -97,6 +97,7 @@ import ListPeopleAndRoles from '@/components/common/ListPeopleAndRoles.vue'
 import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
 import StaffPayment from '@/components/common/StaffPayment.vue'
 import SummaryDefineCompany from '@/components/Incorporation/SummaryDefineCompany.vue'
+import RestorationTypeSummary from '@/components/Restoration/RestorationTypeSummary.vue'
```

Comment:
Is this component supposed to be rendered on this page? I don't see it.

Reply:
I just imported it but I didn't render it since it's going to be a part of a bigger component. The component is going to have other things like Approval type. So I left there to be used later.

Reply:
Isn't it part of your ticket to show restoration/approval type on the review page? If not, do we need another ticket?

Reply:
Yes it is! Okay, I'll add it to the review page. 

![summary](https://user-images.githubusercontent.com/122301442/217396878-024481b6-2ee9-4156-bae3-58a6c8fafa24.PNG)

This is what's going to look like in the end. My summary component is the one in the middle. I'll add and keep it in the review page.